config.yaml

```diff
@@ -73,6 +73,13 @@ repos:
       crate: zincati
       fedora_package: rust-zincati
 
+  console-login-helper-messages:
+    url: https://github.com/coreos/console-login-helper-messages
+    vars:
+      git_repo: console-login-helper-messages
+      fedora_package: console-login-helper-messages
+      rhel9_package: console-login-helper-messages
+
 templates:
```

Comment:
The new template needs to be added here or it'll be ignored.  CI doesn't catch that, unfortunately.

Reply:
https://github.com/coreos/repo-templates/pull/21